adapters/adapter.py

```diff
@@ -485,16 +485,9 @@ def uploadOSMService(self,request):
             LOG.debug(token)
             #file_to_upload = content['service']
             file_to_upload = request
-            file_composed = "@" + file_to_upload
             file = {'nsd-create': open(file_to_upload, 'rb')}           
             data = {'service':file_to_upload}
 
-            HEADERS = {
```

Comment:
@bobeal Are you sure this is not used?

Reply:
yes, headers are passed directly in the curl command that is run some lines below.